package/opt/hassbian/suites/homebridge.sh

```diff
@@ -17,43 +17,49 @@ function homebridge-install-package {
 homebridge-show-long-info
 homebridge-show-copyright-info
 
-echo "Preparing system, and adding dependencies..."
-sudo apt update
-sudo apt -y upgrade
-sudo apt install -y make git
-curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
-sudo apt install -y nodejs
-sudo apt install -y libavahi-compat-libdnssd-dev
-
-echo "Installing homebridge for homeassistant..."
-sudo npm install -g --unsafe-perm homebridge hap-nodejs node-gyp
-sudo npm install -g homebridge-homeassistant
-
-echo "Adding homebridge user, and creating config file..."
-sudo useradd --system --create-home homebridge
-sudo mkdir /home/homebridge/.homebridge
-sudo touch /home/homebridge/.homebridge/config.json
-
 if [ "$ACCEPT" == "true" ]; then
   HOMEASSISTANT_URL="http://127.0.0.1:8123"
   HOMEASSISTANT_PASSWORD=""
 else
   echo ""
+  echo "Please take a moment to setup the Homebridge configuration..."
   echo ""
   echo "Example: https://home.duckdns.org:8123"
   echo -n "Enter your Home Assistant URL and port: "
   read HOMEASSISTANT_URL
   if [ ! "$HOMEASSISTANT_URL" ]; then
       HOMEASSISTANT_URL="http://127.0.0.1:8123"
   fi
-
   echo ""
   echo ""
   echo -n "Enter your Home Assistant API password: "
   read -s HOMEASSISTANT_PASSWORD
   echo
 fi
 
+if [ "$ACCEPT" != "true" ]; then
+  if [ -f "/usr/sbin/samba" ]; then
+    echo -n "Do you want to add samba share for Homebridge configuration? [N/y] : "
+    read SAMBA
+  fi
+fi
+
+echo "Preparing system, and adding dependencies..."
+sudo apt update
+sudo apt -y upgrade
+sudo apt install -y make git
```

Comment:
Isn't `git` and `make` both installed in the base image?

Reply:
They sure are :)